staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -46,6 +49,15 @@ APIS_PKG="$3"
 GROUPS_WITH_VERSIONS="$4"
 shift 4
 
+if [ "${GENS}" = "all" ] || grep -qw "all" <<<"${GENS}"; then
+    ALL="applyconfiguration,client,deepcopy,informer,lister"
+    echo "WARNING: Specifying \"all\" as a generator is deprecated."
+    echo "WARNING: Please list the specific generators needed."
+    echo "WARNING: \"all\" is now an alias for \"${ALL}\""
```

Comment:
maybe call out that it will not include new generators added to this list, like `defaulter`?

Reply:
ACK, will push